REQUIRE

```diff
@@ -1 +1,2 @@
 julia 0.4
+Compat 0.7.14+
```

Comment:
Depends on how many PR's you open, but I don't think the wording in the documentation is what inspires people to use as many trailing `-` and `+` as they do. It's serving as a reminder to yourself not to merge yet, but people may merge sooner without realizing as much.